src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -157,6 +158,45 @@
      */
     List<SimpleInterval> intervalsForTraversal;
 
+    /**
+     * Get the {@link ReferenceDataSource} for this {@link GATKTool}.
```

Comment:
Add disclaimer: "Should only be called by walker base classes in the engine (such as ReadWalker), or by "free-form" tools that extend the GATKTool class directly rather than one of the built-in walker types. Tools that extend a walker type should get their data via apply() rather than directly accessing the engine datasources."

Reply:
Fixed.